bin/transform-gisaid

```diff
@@ -151,9 +152,7 @@ if __name__ == '__main__':
     with open(args.output_fasta, "wt", newline=args.newline) as fasta_fh:
         with open(args.output_additional_info, "wt", newline="") as additional_info_fh, \
              open(args.output_metadata, "wt", newline="") as metadata_fh:
-            dict_writer_kwargs = {}
-            if args.newline is not None:
-                dict_writer_kwargs['lineterminator'] = args.newline
+            dict_writer_kwargs = {'lineterminator': args.newline}
```

Comment:
I actually noticed this earlier.  I believe python's stated reason for doing this is because Excel does not like `.csv` files with just `\n`.  If this is not a concern, we can do this.  If it is a concern, we should go with python's default behavior.

Reply:
Good to know. @emmahodcroft & @eharkins -- if either of you are Excel users, when testing, could you check if you can open the output TSVs from this PR in Excel?

Reply:
So I never normally open up any of the files in Excel. However I've just opened up a `metadata.tsv` with `LF` endings (this is Unix I believe?) and it opened up fine.

Reply:
I never open with Excel either when running the ingest.